src/types/chat/link/interfaces/link.interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface Link {
+  _id: string
+  name: string
+  url: string
+  timestamp: Date
```

Comment:
it will have createdAt and updatedAt properties, so you can extend from CommonEntityFields

export interface Link extends CommonEntityFields {
name: string
url: string

Reply:
> it will have createdAt and updatedAt properties, so you can extend from CommonEntityFields
> 
> export interface Link extends CommonEntityFields { name: string url: string

fixed